.changeset/silly-kids-beg.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fuel-ts/account": patch
+"@fuel-ts/contract": patch
+"fuels": patch
+"@fuel-ts/script": patch
+---
+
+chore: switching `Error` to `FuelError`
```

Comment:
```suggestion
chore: replacing all generic errors with `FuelError`
```

Reply:
If we use `patch` here, it might not make sense to use `chore`

Reply:
It does. For version bumping purposes, most `chores` are patches.

Reply:
@arboleya, `getSystemVersion` uses an error generic error, should I change it too?